src/models/cec-data.ts

```diff
@@ -0,0 +1,13 @@
+import mongoose from 'mongoose';
+import { ICECData, ICECDataMethods, CECDataModel } from '@/types/mongoose/cec-data';
+
+const CECDataSchema = new mongoose.Schema<ICECData, CECDataModel, ICECDataMethods>({
+	creator_pid: Number,
+	game_id: Number,
+	data: String,
+	size: Number,
+	created: BigInt,
+	updated: BigInt
+}, { id: false });
+
+export const CECData: CECDataModel = mongoose.model<ICECData, CECDataModel>('CECData', CECDataSchema);
```

Comment:
We've started to migrate away from unnecessary type defs. This is my fault for introducing them in the first place

```suggestion
export const CECData = mongoose.model<ICECData, CECDataModel>('CECData', CECDataSchema);
```